Message_Spam_Detection/Message_Spam_Detection.py

```diff
@@ -0,0 +1,49 @@
+#importing required libraries
+import pandas as pd
+import string 
+import nltk
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.ensemble import RandomForestClassifier
+import pickle
+import warnings
+warnings.filterwarnings("ignore")
+
+#reading the dataset 
+msg=pd.read_csv("Cleaned_Dataset.csv",encoding='latin-1')
+msg.drop(['Unnamed: 0'],axis=1,inplace=True)
+
+#seperating target and features
+y=pd.DataFrame(msg.label)
+x=msg.drop(['label'],axis=1)
+
+#countvectorization 
+cv=CountVectorizer(max_features=5000)
+temp1=cv.fit_transform(x['final_text'].values.astype('U')).toarray()
+tf=TfidfTransformer()
+temp1=tf.fit_transform(temp1)
+temp1=pd.DataFrame(temp1.toarray(),index=x.index)
+x=pd.concat([x,temp1],axis=1,sort=False)
+
+#drop final_text col
+x.drop(['final_text'],axis=1,inplace=True)
+
+#converting to int datatype
+y=y.astype(int)
+
+#randomforstclassifier model
+model=RandomForestClassifier(n_estimators=100,random_state=0)
```

Comment:
you need to do the model training in the cleaning file only and load the model in this file

Reply:
The trained model has 5000 features (based on the training dataset).
The count vectorizer I used for model training and user input needs to be the same (then only the number of features would be the same)
If I use a different count vectorizer, the number of input features would be different and it will give an error. 
So that's why model training and user input are in same file.

Reply:
okay got it, you can skip this suggestion and work on the previous ones

Reply:
Okay! I have done the required changes!